packages/nimble-components/src/listbox/tests/listbox-matrix.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
+import type { Meta, StoryFn } from '@storybook/html';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../utilities/tests/matrix';
+import { createMatrixThemeStory } from '../../utilities/tests/storybook';
+import { listboxTag } from '..';
+import { listOptionTag } from '../../list-option';
+
+const metadata: Meta = {
+    title: 'Tests/Listbox',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const component = (): ViewTemplate => html`
+    <${listboxTag}>
+        <${listOptionTag} value="1">Option 1</${listOptionTag}>
+        <${listOptionTag} value="2">Option 2</${listOptionTag}>
```

Comment:
Does the listbox need to support disabling an item? If so let's disable one of these to ensure the styling is correct.